src/constants/markets.ts

```diff
@@ -79,9 +84,16 @@ export const MARKET_FILTER_OPTIONS: Record<
   [MarketFilters.ALL]: {
     label: STRING_KEYS.ALL,
   },
+  [MarketFilters.FAVORITED]: {
+    label: undefined, // replace with star icon
+  },
   [MarketFilters.NEW]: {
     label: STRING_KEYS.RECENTLY_LISTED,
   },
+  [MarketFilters.LAUNCHABLE]: {
+    label: STRING_KEYS.LAUNCHABLE,
+    isNew: true,
+  },
```

Comment:
assuming this is the order we want to show this tag in @jaredvu

Reply:
👍